english/marshmallow.md

```diff
@@ -342,7 +342,7 @@ I felt a sudden ease of mind.
 Where is Hermione Granger when you need her? Where I ask you?
 Oh, I know. Click [here](hermione/hermioneHere.md)
 
-You pick up the fork and wrap it in rope to fashion a crappy wand.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.